### PR TITLE
Creating notification list item

### PR DESCRIPTION
**Pull request by Hussomuss**
### **Description**
This PR aims to make a flat list item component for the notifications page to display a variety of items that can be edited.

### **Notification-list-item:**
Used sections to seperate the content , containing left, middle and right section to display different data.

**Utils:**
Created utils folder which contains:

- Props for navigation:

Holds the interface for the navigation props used in more than one file and has been exported for usage

- Time and date helper functions:

Created a helper function to better display the time and day, but also the time ago hours/minutes/days/months ago 

### **Style Changes in other files**
Fixed an issue with the scroll view and be well background padding as it is preferable to use an overall padding. Utilised flat list use of 'horizontal' as it already has scroll view under the hood.

### CODE REVIEW

File: app/homeStyles.ts [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
-`;
+export const Container = styled.TouchableOpacity``;
 
 export const FlatListContainer = styled.View`
   height: fit-content;
 `;
 
+export const FullWidthContainer = styled.View`
+  margin: 0 -12px;
+  width: ${Dimensions.get("window").width}px;
```
Comment (AbbasJay) [inline app/homeStyles.ts:12]: is this needed? can you not just use 100%

File: app/notifications.tsx [hunk 2]
```diff
@@ -1,25 +1,34 @@
...
-      )}
-      {notifications.map((notification) => (
-        <View key={notification.id} style={styles.notification}>
-          <Text>{`Congratulations, your booking is now confirmed. ${notification.createdAt}`}</Text>
+    <BeWellBackground>
+      <NotificationListItem
+        notification={notifications[0]}
+        messageAlert="Booking confirmed!"
+        business={businesses[0]}
+      />
```
Comment (AbbasJay) [inline app/notifications.tsx:16]: map through all notifications

File: app/utils/notification-types.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  userId: number;
+  read: boolean;
+  createdAt: string;
+}
+
+export interface NotificationsContextType {
+  notifications: Notification[];
+  unreadNotificationsCount: number;
+  refreshNotifications: () => void;
+}
```
Comment (AbbasJay) [inline app/utils/notification-types.ts:15]: put this back and export them

File: app/components.tsx [hunk 4]
```diff
@@ -4,6 +4,103 @@ import Button from "./ui/button/button";
 import { BusinessCard } from "./ui/business-card/business-card";
 import { BeWellTabBar } from "@/components/bewellTabBar";
 import { NavigationBar } from "./ui/navigation-bar/navigation-bar";
+import NotificationListItem from "./ui/notification-list-item/notification-list-item";
+import { Business } from "@/app/contexts/BusinessContext";
+import { Notification } from "@/app/utils/notification-types";
+
+const mockBusiness: Business = {
```
Comment (AbbasJay) [inline app/components.tsx:11]: move mock data to a seperate file

File: app/components.tsx [hunk 5]
```diff
@@ -4,6 +4,103 @@ import Button from "./ui/button/button";
...
+  country: "USA",
+  zipCode: "94105",
+  phoneNumber: "(555) 555-5555",
+  email: "info@jabbarfitness.com",
+  type: "Gym",
+  description: "Strength, Conditioning",
+  hours: "Mon-Fri",
+};
+
+const notifications: Array<{
```
Comment (AbbasJay) [inline app/components.tsx:27]: move mock data to a seperate file

File: app/homeStyles.ts (hunk 1 above)
Comment (Hussomuss) [reply app/homeStyles.ts]: auto works equally as well and has now been changed

File: app/components.tsx (hunk 4 above)
Comment (Hussomuss) [reply app/components.tsx]: Done

File: app/components.tsx (hunk 5 above)
Comment (Hussomuss) [reply app/components.tsx]: Done

File: app/homeStyles.ts (hunk 1 above)
Comment (AbbasJay) [reply app/homeStyles.ts]: ok great, can you also remove the import for it if its not being used anywhere else in this file

**Pull request merged**